ios/RNKiwiMobile/RNKiwiViewController.m

```diff
@@ -81,15 +81,73 @@ - (UIStatusBarAnimation)preferredStatusBarUpdateAnimation {
 
 #pragma mark - RNLogger
 
-- (void)ancillaryDisplayed:(NSString *)type {
+- (void)ancillaryDisplayed:(NSString *)type provider:(NSString *)provider {
   if ([self didDisplayAncillary]) {
-    self.didDisplayAncillary(type);
+    self.didDisplayAncillary(type, provider);
   }
 }
 
-- (void)ancillaryPurchased:(NSString *)type {
+- (void)ancillaryPurchased:(NSString *)type provider:(NSString *)provider {
   if ([self didPurchaseAncillary]) {
-    self.didPurchaseAncillary(type);
+    self.didPurchaseAncillary(type, provider);
+  }
+}
+
+- (void)hotelsResultsDisplayed:(id)additionalInfo {
```

Comment:
As discussed below, the signature is different according to the spreadsheet.

Reply:
Changed.